### PR TITLE
[schema] WRED red max threshold and ECN modes

### PR DESCRIPTION
**Pull request by hrachya-mellanox**
* Added support for specifying ECN mode values
* Added red_max_threshold

signed-off-by hrachya@mellanox.com

### CODE REVIEW

File: doc/swss-schema.md [hunk 1]
```diff
@@ -260,14 +260,21 @@ and reflects the LAG ports into the redis under: `LAG_TABLE:<team0>:port`
 	;field						value
 	yellow_max_threshold	= byte_count
 	green_max_threshold		= byte_count
+	red_max_threshold		= byte_count
 	byte_count				= 1*DIGIT
-	
+	ecn						= "ecn_none" / "ecn_green" / "ecn_yellow" / "ecn_red" /  "ecn_green_yellow" / "ecn_green_red" / "ecn_yellow_red" / "ecn_all"
```
Comment (stcheng) [inline doc/swss-schema.md]: change the tabs to spaces?

File: doc/swss-schema.md [hunk 2]
```diff
@@ -260,14 +260,21 @@ and reflects the LAG ports into the redis under: `LAG_TABLE:<team0>:port`
 	;field						value
 	yellow_max_threshold	= byte_count
 	green_max_threshold		= byte_count
+	red_max_threshold		= byte_count
```
Comment (lguohan) [inline doc/swss-schema.md]: we also need the wred_{color}_enable = true /false?

File: doc/swss-schema.md [hunk 3]
```diff
@@ -256,18 +256,34 @@ and reflects the LAG ports into the redis under: `LAG_TABLE:<team0>:port`
...
+	5) "red_max_threshold"
+	6) "1234"
+	7) "ecn"
+	8) "ecn_all"
+	9) "wred_green_enable"
+	10) "true"
+	11) "wred_yellow_enable"
+	12) "true"
+	13) "wred_red_enable"
+	14) "true"
```
Comment (lguohan) [inline doc/swss-schema.md]: tab to spaces

**Comment by lguohan** [comment]
> 

all tabs to spaces.

---
Refers to: doc/swss-schema.md:309 in d74b970. [](commit_id = d74b970ca05e8a1c81b138ce41b8c14b1cbe8453, deletion_comment = False)

**Comment by hrachya-mellanox** [comment]
converted all tabs to spaces

**Comment by lguohan** [comment]
:shipit:

**Pull request merged**